gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/DbDataStoreTestsUtils.js

```diff
@@ -82,45 +82,57 @@ gpii.tests.dbDataStore.verifyFetched = function (response, expected) {
     jqUnit.assertLeftHand("The data is saved successfully", expected, response);
 };
 
+gpii.tests.dbDataStore.verifyFetchedGpiiAppInstallationAuthorization = function (response, dataToSave) {
```

Comment:
I think that `expected` might be clearer than `dataToSave` (and will match `gpii.tests.dbDataStore.verifyFetched` which we partially delegate to).